.editorconfig

```diff
@@ -95,6 +95,9 @@ dotnet_diagnostic.CA2016.severity = error
 dotnet_diagnostic.CA2211.severity = error
 dotnet_diagnostic.CA2241.severity = error
 
+# Temporarly diabled to fix the tech dept.
```

Comment:
Temporar[i]ly di[s]abled

Reply:
Issue for this to re-enable?

Reply:
Just too many violations, which would muddle this PR

Reply:
Issue #439 created